File: kotlin/sprint0/D.kt
private fun getTwoSum(a: List<Int>, k: Int): Pair<Int, Int>? {
    TODO()
    // Your code
    // ヽ(´▽`)/
}

fun main() {
    val size = readInt()
    val a = readInts()
    val k = readInt()
    val result = getTwoSum(a, k)
    if (result == null) {
        println("None")
    } else {
        println("${result.first} ${result.second}")
    }
}

private fun readStr() = readln()
private fun readInt() = readStr().toInt()
private fun readStrings() = readStr().split(" ")
private fun readInts() = readStrings().map { it.toInt() }

File: swift/sprint0/D.swift

func getTwoSum(_ a: Array<Int>, _ target: Int) -> (Int, Int)? {
    // Your code
    // “ヽ(´▽｀)ノ”
}

func readInts() -> Array<Int> {
    return readLine()!.split(separator: " ").map { (x: Substring) -> Int in
        return Int(x)!
    }
}

func printArray<T>(_ a: Array<T>) {
    for element in a {
        print(element, terminator: " ")
    }
}

let size = Int(readLine()!)!
let a = readInts()
let target = Int(readLine()!)!
let pair = getTwoSum(a, target)
if (pair == nil) {
    print("None")
} else {
    print(pair!.0, pair!.1, terminator: " ")
}

